components/generic/CompareImageSlider.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <figure
+    ref="container"
+    class="compare-image"
+    data-qa="compare image"
+  >
+    <img
+      ref="imageLeft"
+      :src="imageLeft"
+      :alt="imageLeftText"
+      class="image-left"
+      :style="leftImageClip"
+      data-qa="compare image left image"
+    >
+    <img
+      ref="imageRight"
+      :src="imageRight"
+      :alt="imageRightText"
+      class="image-right"
+      data-qa="compare image right image"
+    >
+    <div
+      ref="slider"
+      class="slider"
+      :style="sliderBarPosition"
+      data-qa="compare image slider"
+      @mousedown="initDrag"
+    >
+      <span class="slider-bar" />
+      <button
+        :class="{ 'is-active' : dragging }"
+        class="slider-handle"
+        data-qa="compare image slider handler"
+      >
+        <span class="sr-only">Slider Handle</span>
+      </button>
+    </div>
+  </figure>
```

Comment:
I am not sure if it is part of this task, so it might be worth discussing with PO, but I can imagine that there is a paragraph with description here as well. That says something like: "On the left, you see Person A in here twenties. On the right, you can see Person A in here fifties".

Reply:
This can be done in a different ticket as its not part of the criteria